### DISCUSSION TITLE
Where are located the pages Log in and Register?

### DISCUSSION THREAD

**Discussion by HeyBaldur** [jasontaylordev/CleanArchitecture#1103] [General]
I have been searching through the whole app the **Register** and **Log In** but no luck! I just want to make some changes in the UI but it is very difficult to find these pages. Can anyone please tell me where these pages are?

**Comment by devlear** [comment]
I was wondering the same thing. After looking around for a bit today, it looks like it is auto generated from the Microsoft.AspNetCore.Identity. The class description for .AddDefaultIdentity<ApplicationUser>() indicated that it will auto-build the ui.

Reference: https://learn.microsoft.com/en-us/aspnet/core/security/authentication/identity?view=aspnetcore-8.0&tabs=visual-studio
More References: https://stackoverflow.com/questions/50802781/where-are-the-login-and-register-pages-in-an-aspnet-core-scaffolded-app

**Reply by HeyBaldur** [reply]
Thanks for sharing, I personally don't like this approach of using Microsoft.AspNetCore.Identity for creating UI pages, specially if this project already has an **Angular UI**. _It is my opinion._
